sdk/storage/Azure.Storage.Queues/CHANGELOG.md

```diff
@@ -1,14 +1,13 @@
 # Release History
 
-## 12.23.0-beta.2 (Unreleased)
+## 12.24.0-beta.1 (Unreleased)
 
 ### Features Added
+- Added support for service version 2025-11-05.
+- Added more useful error message when the SDK encounters an x-ms-version mis-match issue.
 
 ### Breaking Changes
-
-### Bugs Fixed
-
-### Other Changes
+- Added `QueueProperties.ApproximateMessagesCountLong` to replace `QueueProperties.ApproximateMessagesCount`
```

Comment:
Is this a breaking change? I thought `QueueProperties.ApproximateMessagesCount` was just being deprecated (e.g. EB(none)), not being removed.

I think this should be moved to Features Added or Other Changes and be reworded something like "Added `QueueProperties.ApproximateMessagesCountLong` to support values larger than `int` can support in `QueueProperties.ApproximateMessagesCount`

Reply:
Agreed, moved to Features Added.